.travis.yml

```diff
@@ -12,7 +12,7 @@ branches:
 jobs:
   include:
     # Quickly run tests in dev mode.
-    - rust: stable
+    - rust: 1.32.0
```

Comment:
Can we add a comment here that this should be kept in sync with the `rust-toolchain` file and the Dockerfile? It could say something like "rather than updating this manually, run `update-rust-version.sh`...

Reply:
added :) also added to the Dockerfile